buddyboss/buddyboss-platform
# Fixed #2946 Removed additional space after mention @name

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #2946 .

### How to test the changes in this Pull Request:

1.Type the mention with the @ in the News Feed
2.Click on any name on the suggession list, you will see one space now after that mention

### Proof Screenshots or Video before/after fix

http://somup.com/cr6UQb3Ekx
http://somup.com/cr6UQO3Epb

### Other information:




### Changelog entry

> Removed additional space after mention @name
